vpnglean/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+add_library(vpnglean STATIC IMPORTED GLOBAL)
+
+set_target_properties(vpnglean PROPERTIES FOLDER "Libs")
+
+if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
+    set(OPENSSL_USE_STATIC_LIBS TRUE)
+    find_package(OpenSSL REQUIRED)
+    cmake_path(GET OPENSSL_SSL_LIBRARY PARENT_PATH OPENSSL_LIB_DIR)
+endif()
+
+set(CARGO_BUILD_ENV
+    BUILD_ID=${BUILD_ID}
+    APP_VERSION=${APP_VERSION}
```

Comment:
I don't think `APP_VERSION` exists as a CMake variable, did you want `${CMAKE_PROJECT_VERSION}` instead?

Reply:
Ha, yeap you are right. Why did I think it existed?! :eyes: :exploding_head: